pkg/controllers/vdb/sandboxsubcluster_reconciler.go

```diff
@@ -173,6 +173,12 @@ func (s *SandboxSubclusterReconciler) executeSandboxCommand(ctx context.Context,
 			sb, found := scSbMap[sc]
 			if !found {
 				// assume it is already sandboxed
+				if j == 0 {
+					// we need the sandbox primary sc to be up so we can tell it about the sc being added
+					if res, err := s.waitForSandboxScUp(ctx, vdbSb.Name, sc); err != nil {
+						return res, err
```

Comment:
Rather than returning an error, we should return a "ctrl.Result{Requeue: true}, nil" since error process is expensive in K8s.